test/NuGet.Core.Tests/NuGet.Protocol.Tests/Resources/RepositorySignatureResourceTests.cs

```diff
@@ -26,13 +26,15 @@ public class RepositorySignatureResourceTests
         private const string _notBefore = "2018-02-26T00:00:00.0000000Z";
 
         [Fact]
-        public async Task RepositorySignatureResource_Basic()
+        public async Task RepositorySignatureResource_BasicAsync()
         {
             // Arrange
-            var source = $"http://unit.test/{Guid.NewGuid()}/v3-with-flat-container/index.json";
-            var responses = new Dictionary<string, string>();
-            responses.Add(source, JsonData.RepoSignIndexJsonData);
-            responses.Add("https://api.nuget.org/v3-index/repository-signatures/index.json", JsonData.RepoSignData);
+            var source = $"https://unit.test-{Guid.NewGuid()}/v3-with-flat-container/index.json";
```

Comment:
Ideally, the root domain would be [`.test`](https://en.wikipedia.org/wiki/.test) and the GUID would be added as or to a subdomain.  For example:

    $"https://{Guid.NewGuid()}.unit.test/v3-with-flat-container/index.json";